### PR TITLE
Adding N2WS Backup and Recovery integration

### PR DESCRIPTION
**Pull request by eliadeini**
Adding N2WS Backup and Recovery integration to DataDog

### What does this PR do?

Adds N2WS Backup & Recovery (CPM) integration to DataDog

### Motivation

N2WS Backup & Recovery supports from version 3.2 and above communication with DataDog agent. This PR is a part of a process of adding an N2WS tile.

### Review checklist

- [ ] PR has a [meaningful title](https://github.com/DataDog/integrations-core/blob/master/CONTRIBUTING.md#pull-request-title) or PR has the `no-changelog` label attached
- [ ] Feature or bugfix has tests
- [ ] Git history is clean
- [ ] If PR impacts documentation, docs team has been notified or an issue has been opened on the [documentation repo](https://github.com/DataDog/documentation/issues/new)

### Additional Notes

Anything else we should know when reviewing?

### CODE REVIEW

File: .gitignore
Comment (hithwen) [inline .gitignore:121]: Please, do not add these to the global git ignore. Specially the generic *.py and *.yaml

**Review by hithwen — DISMISSED** [review]
The changes from https://github.com/DataDog/integrations-extras/pull/753 need to be included on this PR, not in a separate one.

File: n2ws/README.md
Comment (hithwen) [inline n2ws/README.md:55]: I see this PR includes some dashboards. Once this PR is merged there will be a tile in app for N2WS integration too. If users click install on it, then the dashboards will get installed on their account.

File: .github/CODEOWNERS
Comment (hithwen) [inline .github/CODEOWNERS:37]:
```suggestion
/n2ws/                  @eliadeini eliad.eini@n2ws.com
```

File: n2ws/manifest.json
Comment (hithwen) [inline n2ws/manifest.json:9]: Can you keep this under 80 chars?

**Comment by hithwen** [comment]
Can you add a metadata.csv file with the expected metrics that this integration is going to emit?

File: n2ws/manifest.json
Comment (hithwen) [inline n2ws/manifest.json:7]: This should contain, if possible, a metric that is always emitted.

File: n2ws/manifest.json
Comment (hithwen) [inline n2ws/manifest.json:6]: Is there a way to configure the prefix in N2WS?

**Comment by hithwen** [comment]
Also can you ensure that the README follows the same format as others and includes the same sections? (Metrics, Service checks etc). The readme contents are used to render the in-app tile

**Comment by kayayarai** [comment]
Opened DOCS-1367 to track edit of user-facing content in this PR.

**Comment by eliadeini** [comment]
> Opened DOCS-1367 to track edit of user-facing content in this PR.

Hi, is there another thing I should do now in order to merge this PR?

**Comment by hithwen** [comment]
> Hi, is there another thing I should do now in order to merge this PR?

Just wait for the editorial review

File: n2ws/manifest.json
Comment (hithwen) [inline n2ws/manifest.json:31]:
```suggestion
    "metrics_metadata": "metadata.csv",
```

File: n2ws/README.md
Comment (cswatt) [inline n2ws/README.md:39]: Instead of walking through that whole thing, let's just say:

Connect to your N2WS Backup and Recovery instance with SSH and run the following command to install the Datadog Agent. You may require `sudo` privileges to perform this action: `<insert command here>`

File: n2ws/README.md
Comment (cswatt) [inline n2ws/README.md:43]: Rather than navigating through, let's just link to the Metrics Explorer - https://app.datadoghq.com/metric/explorer

File: n2ws/README.md
Comment (cswatt) [inline n2ws/README.md:50]:
```suggestion
**Graph**: Select your metric from the list. All N2WS metrics begin with the string ‘cpm_metric’.

**Over**: Select data from the list. All N2WS user data begins with the string ‘cpm:user:<user-name>’. You can select either a specific user or the entire N2WS instance.
```

File: n2ws/README.md
Comment (cswatt) [inline n2ws/README.md:54]: Just link to the integration rather than giving directions.

File: n2ws/README.md
Comment (cswatt) [inline n2ws/README.md:58]:
```suggestion
Alternatively, users can [import JSON templates from N2WS][link here].
```

put links at bottom of this file (see other integrations readmes for how we typically do this)

File: n2ws/README.md
Comment (cswatt) [inline n2ws/README.md:63]:
```suggestion
Datadog collects data about N2WS Backup & Recovery backups: number of snapshots of each type, successful backups, failed backups, partially successful backups, 
```

File: n2ws/README.md
Comment (cswatt) [inline n2ws/README.md:64]:
```suggestion
protected resources from any type, data about volume capacity, alerts, etc.
```

File: n2ws/README.md
Comment (cswatt) [inline n2ws/README.md:69]: Missing the link here.

File: n2ws/README.md
Comment (cswatt) [inline n2ws/README.md:86]: Let's remove this section. We assume that users are N2WS users and can access the documentation and support team themselves. Unless there is something of note that integration users should specifically be aware of in the N2WS docs, we don't need to link them.

**Review by cswatt — CHANGES_REQUESTED** [review]
styleguide changes to the documentation

File: n2ws/README.md
Comment (eliadeini) [reply n2ws/README.md:58]: Will it become a clickable text?

Comment (hithwen) [reply n2ws/README.md:58]: yes

File: n2ws/README.md
Comment (eliadeini) [reply n2ws/README.md:26]: Does this link valid for all DataDog accounts regardless of regional location?

File: n2ws/README.md
Comment (eliadeini) [reply n2ws/README.md:33]: Using back-ticks `` , right?

File: n2ws/README.md
Comment (eliadeini) [reply n2ws/README.md:39]: Every DataDog account has a different command, that is why we can't insert a command here.

File: n2ws/README.md
Comment (eliadeini) [reply n2ws/README.md:54]: I guess that the link will look like that: https://app.datadoghq.com/account/settings#integrations/N2WS

If it will look different, please tell us

Comment (eliadeini) [reply n2ws/README.md:54]: In addition, I think that the user is already in integration tile when he reads this text, isn't he?

File: n2ws/README.md
Comment (eliadeini) [reply n2ws/README.md:86]: Since we are considering getting to DataDog's marketplace in the future, the users might be DataDog's customers that will need this link.
I would keep it as is.

**Review by hithwen — APPROVED** [review]

**Pull request merged**
